python 3.6 sqlite3 bug

```
>       cur.execute("""VACUUM;""")
E       sqlite3.OperationalError: cannot VACUUM from within a transaction
```

Tests work with python 3.5.

This is a known issue with python 3.6: https://bugs.python.org/issue28518 - The fix will likely land in Python 3.6.2. In the meantime, it blocks py36 support 😐 

/cc @sgillies

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,6 +2,7 @@ language: python
 python:
   - 2.6
   - 2.7
+  - 3.6
 install:
   - pip install nose
 script:

--- a/mbutil/util.py
+++ b/mbutil/util.py
@@ -71,7 +71,13 @@ def optimize_database(cur, silent):
     cur.execute("""ANALYZE;""")
     if not silent: 
         logger.debug('cleaning db')
+
+    # Workaround for python>=3.6.0,python<3.6.2
+    # https://bugs.python.org/issue28518
+    cur.isolation_level = None
     cur.execute("""VACUUM;""")
+    cur.isolation_level = ''  # reset default value of isolation_level
+
 
 def compression_do(cur, con, chunk, silent):
     if not silent:

